app/src/main/java/com/pvp/app/ui/screen/calendar/DateUtil.kt

```diff
@@ -0,0 +1,37 @@
+package com.pvp.app.ui.screen.calendar
+
+import java.time.DayOfWeek
+import java.time.LocalDate
+import java.time.YearMonth
+import java.time.format.TextStyle
+import java.util.Locale
+
+object DateUtil {
+
+    val daysOfWeek: Array<String>
+        get() {
+            val daysOfWeek = Array(7) { "" }
+
+            for (dayOfWeek in DayOfWeek.entries) {
+                val localizedDayName =
+                    dayOfWeek.getDisplayName(TextStyle.SHORT, Locale.getDefault())
```

Comment:
Multiple parameters could be split.

Also, `localizedDayName` is resolved quite simply, I believe disposing the single-referenced variable altogether would still keep the code quite readable and _somewhat_ of a problem regarding redundant variable initialization would be solved. At least, I think that it is redundant here.
```kotlin
daysOfWeek[dayOfWeek.value - 1] = dayOfWeek.getDisplayName(
    TextStyle.SHORT,
    Locale.getDefault()
)
```